src/utils/MultiInviter.ts

```diff
@@ -168,10 +168,18 @@ export default class MultiInviter {
             }
 
             if (!ignoreProfile && SettingsStore.getValue("promptBeforeInviteUnknownUsers", this.roomId)) {
-                const profile = await this.matrixClient.getProfileInfo(addr);
-                if (!profile) {
-                    // noinspection ExceptionCaughtLocallyJS
-                    throw new Error("User has no profile");
+                try {
+                    await this.matrixClient.getProfileInfo(addr);
+                } catch (err) {
+                    // convert the errors so that they can be handled correctly later on
+                    switch (err.errcode) {
+                        case 'M_FORBIDDEN':
```

Comment:
Might be nice to add a comment explaining why this maps to undisclosed

Reply:
I hope the comment is better now.